### PR TITLE
[Topic Entry] MySQL

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description

Added a Topic Entry for MySQL

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/mysql/mysql.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+Title: 'MySQL'
+Description: 'MySQL is an open-source relational database management system (RDBMS) used for managing and organizing data with SQL.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/mysql'
+CatalogContent:
+  - 'getting-started-off-platform-for-data-science'
+  - 'paths/design-databases-with-mysql'
+---
+
+MySQL is an open-source [relational database](https://www.codecademy.com/resources/docs/general/database/relational-database) management system developed by MySQL AB and now owned by Oracle Corporation. It uses [SQL](https://www.codecademy.com/resources/docs/sql) to manage and organize data into related tables, supporting high performance, scalability, and robust security features. It is extensively used in web applications, data warehousing, e-commerce, logging, and content management systems like WordPress. Its flexibility and efficiency make it ideal for both small-scale projects and large-scale, data-intensive applications used by major companies like Facebook and YouTube.
```
Comment (cigar-galaxy82) [inline content/mysql/mysql.md:10]:
```suggestion
**MySQL** is an open-source [relational database](https://www.codecademy.com/resources/docs/general/database/relational-database) management system developed by MySQL AB and now owned by Oracle Corporation. It uses [SQL](https://www.codecademy.com/resources/docs/sql) to manage and organize data into related tables, supporting high performance, scalability, and robust security features. It is extensively used in web applications, data warehousing, e-commerce, logging, and content management systems like WordPress. Its flexibility and efficiency make it ideal for both small-scale projects and large-scale, data-intensive applications used by major companies like Facebook and YouTube.
```

**Comment by cigar-galaxy82** [comment]
Hey @mamtawardhani 
Thank you for contributing to Codecademy Docs! The entry is well written!

I've suggested a few changes modify them at your convenience

Thank you!

**Comment by mamtawardhani** [comment]
Its done @cigar-galaxy82 👍

**Comment by mamtawardhani** [comment]
@cigar-galaxy82 you need to add the `status: ready for next review` so that it gets assigned to another reviewer for the second review. :)

**Review by Sriparno08 — APPROVED** [review]
Looks good, @mamtawardhani! 👍

**Pull request merged**